layer/index.js

```diff
@@ -20,8 +20,7 @@ router.route('/:layerId')
         try {
             const layer = await findLayerById(layerId, projectId, true)
             if (!layer) {
-                respondWithError(res, 404, 'No layer found with that ID.')
-                return
+                return respondWithError(res, 404, 'No layer found with that ID.')
```

Comment:
So, the reason for this pattern is because `respondWithError()` doesn't return anything. It sends a res.json(), which is bypassing the normal express handling. We do this in other places where we do `next() /n return` or just `res.json(data)`. In reality the return should be redundant, but it is useful in preventing execution in conditionals and guard clauses where we don't want to continue.